Set lowest supported KGP and AGP versions to ones not using conventions for Gradle 9.0 (#32720)
gradle/gradle: platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

=== Potential breaking changes

==== Lowest supported Kotlin Gradle Plugin version change

> Starting with Gradle 9.0 the lowest supported Kotlin Gradle Plugin version is 1.9.21.

Reviewer: ```suggestion
Starting with Gradle 9.0, the minimum supported Kotlin Gradle Plugin version is 1.9.21.
```

> Previous versions are not supported anymore because they use now removed Gradle APIs.

Reviewer: ```suggestion
Earlier versions are no longer supported as they rely on Gradle APIs that have been removed.
```

> The lowest supported Kotlin Gradle Plugin version for Gradle 8.x was 1.6.10.

Reviewer: ```suggestion
For Gradle 8.x, the minimum supported version was 1.6.10.
```

==== Lowest supported Android Gradle Plugin version change

> Starting with Gradle 9.0 the lowest supported Android Gradle Plugin version is 7.4.0.

Reviewer: ```suggestion
Starting with Gradle 9.0, the minimum supported Android Gradle Plugin version is 7.4.0.
```

> Previous versions are not supported anymore because they use now removed Gradle APIs.

Reviewer: ```suggestion
Earlier versions are no longer supported as they rely on Gradle APIs that have been removed.
```

> The lowest supported Android Gradle Plugin version for Gradle 8.x was 7.3.0.

Reviewer: ```suggestion
For Gradle 8.x, the minimum supported version was 7.3.0.
```
